CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.6.0] - 2020-01-29
+### Changed
+- On Sqlalchemy allow to use choices on argument parsing
```

Comment:
The list of changes should be exhaustive, can you please add the fact that the GET, DELETE parsers and POST and PUT models can now check the restricted values. Also a note about validation inside models would be nice :)

Reply:
-- move to unreleased